apps/snippets/infra/global/lambda.py

```diff
@@ -0,0 +1,15 @@
+def lambda_handler(event, context):
+     response = event['Records'][0]['cf']['response']
```

Comment:
This is a Origin Response example, that's why it wasn't working. I've made some changes via the console testing out some things. Once I get comfortable with lambda@edge i'll update terraform

Reply:
that's what I get for copy/pasting code :-) thanks for looking into this @glogiotatidis!